src/aliceVision/fuseCut/DelaunayGraphCut.cpp

```diff
@@ -1629,7 +1629,7 @@ DelaunayGraphCut::GeometryIntersection DelaunayGraphCut::rayIntersectTriangle(co
     Point3d tempIntersectPt;
     const Point2d triangleUv = getLineTriangleIntersectBarycCoords(&tempIntersectPt, A, B, C, &originPt, &DirVec);
 
-    if (!isnormal(tempIntersectPt.x) || !isnormal(tempIntersectPt.y) || !isnormal(tempIntersectPt.z))
+    if (!std::isnormal(tempIntersectPt.x) || !std::isnormal(tempIntersectPt.y) || !std::isnormal(tempIntersectPt.z))
```

Comment:
Shouldn't we `#include <cmath>` so that we are sure it is included and thus we do not rely on other includes to import the function?

Reply:
I agree. This is fixed now.